packages/webpack-less/extract.js

```diff
@@ -3,8 +3,7 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const isProduction = process.env.NODE_ENV === 'production';
 
 const plugin = new MiniCssExtractPlugin({
-  filename: isProduction ? '[name].min.css' : '[name].css',
-  disable: !isProduction
```

Comment:
Is it not necessary to disable the plugin anymore?

Reply:
The new version does not have `disable` flag. I have tested dev and pro, and it does not throw any error.